docs/reference/fleet/apis/fleet-apis.asciidoc

```diff
@@ -0,0 +1,13 @@
+[role="xpack"]
```

Comment:
The preview diff from the docs build did not include this. I think you need to add it to `index.asciidoc` too to get it on the main page and thus included into the docs output.

Reply:
I still do not see it in the preview. I think a link is still missing here:

https://github.com/elastic/elasticsearch/blob/2657496dc72f900fc901f4650bd52e7eb1f17da9/docs/reference/rest-api/index.asciidoc#L50

similar to the one for CCR.